clients/src/main/java/org/apache/kafka/clients/admin/ConsumerGroupDescription.java

```diff
@@ -40,15 +40,6 @@
     private final Node coordinator;
     private final Set<AclOperation> authorizedOperations;
 
-    public ConsumerGroupDescription(String groupId,
```

Comment:
This is a public constructor, I'm not entirely sure we can remove it without a deprecation